elliott/elliottlib/cvp.py

```diff
@@ -55,6 +55,21 @@ async def latest_sanity_test_results(self, nvrs: Iterable[str]) -> Dict[str, Opt
             nvr_results[nvr] = None  # missing result
         return nvr_results
 
+    async def latest_bundle_test_results(self, bundle_nvrs: Iterable[str]) -> Dict[str, Optional[Dict]]:
+        """ Get latest CVP test results for specified bundle build NVRs
```

Comment:
In this function, we want to call this url for the nvrs that are passed in http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/ 
then parse the results and return